packages/backend/src/payment-method/ilp/spsp/middleware.ts

```diff
@@ -24,11 +37,20 @@ const spspMiddleware = async (
 ): Promise<void> => {
   // Fall back to legacy protocols if client doesn't support Open Payments.
   if (ctx.accepts('application/spsp4+json')) {
-    const receiver = ctx.walletAddress ?? ctx.incomingPayment
```

Comment:
`ctx.incomingPayment` was a leftover thing from when we had the "connection" resource in Open Payments. does not apply anymore